pkgs/applications/version-management/git-and-tools/conform/default.nix

```
@@ -0,0 +1,24 @@
+{ pkgs, lib, buildGoModule, fetchFromGitHub }:
```

```suggestion
{ lib, buildGoModule, fetchFromGitHub, git }:
```
we don't need to use that here.